src/main/resources/net/minecraft/fukkit.block.mixins.json

```diff
@@ -2,6 +2,7 @@
   "required": true,
   "package": "com.github.fukkitmc.fukkit.mixin.net.minecraft.block",
   "compatibilityLevel": "JAVA_8",
+  "plugin": "com.github.fukkitmc.fukkit.mixin.plugin.CraftFukkitMixinPlugin",
```

Comment:
You seem to have specified the mixin plugin in 4 mixin configs. Half should look at this